src/error.rs

```diff
@@ -100,7 +100,7 @@ impl From<RuntimeError> for InterpreterError {
 }
 
 #[derive(Debug, Error)]
-#[cfg_attr(feature = "serde-types-minimal", derive(serde::Serialize, serde::Deserialize))]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
I think `cargo test --features serde` fails with `the trait "Serialize" is not implemented for "std::io::Error"` and `the trait "Deserialize<'_>" is not implemented for "std::io::Error"`

Reply:
And as far as I can see CI does not test for feature `serde`